Installation/cmake/modules/CGAL_Macros.cmake

```diff
@@ -287,7 +287,9 @@ if( NOT CGAL_MACROS_FILE_INCLUDED )
 
         if (OPENGL_FOUND)
           add_definitions("${OPENGL_DEFINITIONS}")
-          include_directories( SYSTEM "${OPENGL_INCLUDE_DIR}" )
+          if ( NOT "${OPENGL_INCLUDE_DIR}" STREQUAL "" )
```

Comment:
The condition can be simplified to `if(OPENGL_INCLUDE_DIR)` and let CMake decide if the variable has a correct content.

What is more, in which case can we have `OPENGL_FOUND` but not `OPENGL_INCLUDE_DIR`?